Tasks/NodeTool/nodetool.ts

```diff
@@ -27,149 +32,136 @@ async function run() {
 //
 interface INodeVersion {
     version: string,
-    lts: any,
     files: string[]
 }
 
 //
 // Basic pattern:
-//     version = find and evaluate local versions
-//          use latest match 
-//     if version not found locally
-//          // let's query
-//          if versionSpec is explicit version
-//               versionToGet = versionSpec
-//          else
-//               versionToGet = query and evaluate internet tool provider
-//          
-//          download versionToGet
-//          Extract or move to cache download
-//
-//      find tool path by version
-//      prepend $PATH with toolpath
+//      if !checkLatest
+//          toolPath = check cache
+//      if !toolPath
+//          if version is a range
+//              match = query nodejs.org
+//              if !match
+//                  fail
+//              toolPath = check cache
+//          if !toolPath
+//              download, extract, and cache
+//              toolPath = cacheDir
+//      PATH = cacheDir + PATH
 //
-async function getNode(versionSpec: string, onlyLTS: boolean) {
-    //
-    // Let's try and resolve the versions spec locally first
-    //
-    let localVersions: string[] = toolLib.findLocalToolVersions('node');
-    let version: string = toolLib.evaluateVersions(localVersions, versionSpec);
+async function getNode(versionSpec: string, checkLatest: boolean) {
```

Comment:
There are a few significant changes in this PR:
1. Deprecated LTS Only
1. Add Check Latest
1. Fix bug where user enters node version without "v" prefix and building the URL fails. To fix this, I added `cleanVersion(version: string): string` to the lib. And then when building the URL I added "v" into the format string and append the cleaned version string.
1. Identify places where logging (debug or output) belongs in the lib. I wanted to reduce the burden on the caller. For example, I feel like when downloading a file, then we should always log-output `Downloading <URL>`. It could take time, and communication to the user is warranted.